cumulusci/core/dependencies/resolvers.py

```diff
@@ -402,13 +402,29 @@ def get_resolver_stack(
     raise CumulusCIException(f"Resolver stack {name} was not found.")
 
 
+def dependency_filter_ignore_deps(ignore_deps: List[dict]) -> Callable:
+    ignore_github = [d["github"] for d in ignore_deps if "github" in d]
+    ignore_namespace = [d["namespace"] for d in ignore_deps if "namespace" in d]
+
+    def should_include(some_dep: Dependency) -> bool:
+
+        if isinstance(some_dep, PackageNamespaceVersionDependency):
+            return some_dep.namespace not in ignore_namespace
+        if isinstance(some_dep, BaseGitHubDependency):
+            return some_dep.github not in ignore_github
```

Comment:
This looks like something that could maybe be refactored into a method on Dependency classes, but it's also simple enough as is that I wouldn't necessarily rush to do that.